app/views/content/train-to-be-a-teacher/promos/_find-your-ske-course.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="find-your-course">
+  <div class="left">
+    <div class="icon">
+       <%= image_pack_tag("media/images/icon-search-black.svg", alt: "") %>
+    </div>
+  </div>
+
+  <div class="right">
+    <h2 class="heading-m heading--margin-top-0">Find your SKE course</h2>
+    <p>
+      Take a look at the different SKE courses available.
+    </p>
+
+    <p>
+      <a class="button" href="https://www.gov.uk/government/publications/subject-knowledge-enhancement-course-directory/subject-knowledge-enhancement-ske-course-directory">Search for a course</a>
```

Comment:
```suggestion
      <a class="button" href="https://www.gov.uk/government/publications/subject-knowledge-enhancement-course-directory/subject-knowledge-enhancement-ske-course-directory">Search for a SKE course</a>
```
should we add `SKE` to the link text, to make it different from other Find buttons @EllieNodder ? might be useful for GA tracking to distinguish the links from those on other pages?